www/docs/server/context.md

```diff
@@ -67,4 +67,17 @@ export const publicProcedure = t.procedure;
  * Protected procedure
  **/
 export const protectedProcedure = t.procedure.use(isAuthed);
-```
\ No newline at end of file
+
+// -------------------------------------------------
+// @filename: [trpc].ts
+// -------------------------------------------------
+import * as trpcNext from '@trpc/server/adapters/next';
+import { appRouter } from '../../../server/routers/_app';
+import { createContext } from '../../../server/context';
```

Comment:
```suggestion
import { appRouter } from './_app';
import { createContext } from './context';
```

Reply:
I've made this change. But I don't think this is correct, since this would be inside `[...trpc].ts` inside NextJS routes we would need to `../../` couple of times to reach the `server` directory